whois-api/src/main/java/net/ripe/db/whois/api/rdap/VCardBuilder.java

```diff
@@ -152,4 +154,8 @@ public VCardBuilder addUrl(final Map parameters, final String value) {
     public VCard build() {
         return new VCard(properties);
     }
+
+    private List getAdrTextOrDefault(List values){
+        return isEmpty(values) ? nCopies(7, "") : values; //Vcard text format is 7 empty element
```

Comment:
Maybe use a default array, so it's not re-created every time (unless it's later modified?).

```
private static final List<String> DEFAULT_VCARD = Collections.unmodifiableList(Lists.newArrayList("", "", "", "", "", "", ""));
```

Reply:
Can't use a static List (returned list can be modified)